modules/juce_audio_formats/codecs/juce_WavPackAudioFormat.cpp

```diff
@@ -0,0 +1,315 @@
+/*
+  ==============================================================================
+
+   This file is part of the JUCE library.
+   Copyright (c) 2015 - ROLI Ltd.
+
+   Permission is granted to use this software under the terms of either:
+   a) the GPL v2 (or any later version)
+   b) the Affero GPL v3
+
+   Details of these licenses can be found at: www.gnu.org/licenses
+
+   JUCE is distributed in the hope that it will be useful, but WITHOUT ANY
+   WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+   A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+   ------------------------------------------------------------------------------
+
+   To release a closed-source product which uses JUCE, commercial licenses are
+   available: visit www.juce.com for more information.
+
+  ==============================================================================
+*/
+
+#if JUCE_USE_WAVPACK
+
+namespace WavPackNamespace
+{
+#include <wavpack/wavpack.h>
```

Comment:
Yes, we don't generally include 3rd party headers unless really necessary. What is in wavpack.h and where would someone get it? Does it rely on a static library to go with it, or is it header only?